content/manuals/security/for-admins/hardened-desktop/settings-management/_index.md

```diff
@@ -50,33 +48,27 @@ Using the `admin-settings.json` file, admins can:
 - Control whether developers can use the Docker terminal
 - Control the file sharing implementation for your developers on macOS
 - Specify which paths your developers can add file shares to
-- Configure Air-Gapped Containers
-
-For more details on the syntax and options admins can set, see [Configure Settings Management](configure.md).
+- Configure Air-gapped containers
 
-### How do I set up and enforce Settings Management?
+For more details on the syntax and options, see [Configure Settings Management](configure.md).
 
-As an administrator, you first need to [enforce
-sign-in](/manuals/security/for-admins/enforce-sign-in/_index.md). This is
-because the Settings Management feature requires a Docker Business subscription
-and therefore your Docker Desktop developers must authenticate to your
-organization. Enforcing sign-in ensures that your Docker Desktop developers
-always authenticate to your organization, even though they can authenticate
-without it and the feature will take effect. Enforcing sign-in guarantees the
-feature always takes effect.
+## How do I set up and enforce Settings Management?
 
+You first need to [enforce sign-in](/manuals/security/for-admins/enforce-sign-in/_index.md) to ensure that all Docker Desktop developers authenticate with your organization. Since the Settings Management feature requires a Docker Business subscription, enforced sign-in guarantees that only authenticated users have access and that the feature consistently takes effect across all users, even though it may still work without enforced sign-in.
 
 Next, you must either manually [create and configure the admin-settings.json file](configure.md), or use the `--admin-settings` installer flag on [macOS](/manuals/desktop/setup/install/mac-install.md#install-from-the-command-line) or [Windows](/manuals/desktop/setup/install/windows-install.md#install-from-the-command-line) to automatically create the `admin-settings.json` and save it in the correct location.
 
 Once this is done, Docker Desktop developers receive the changed settings when they either:
 - Quit, re-launch, and sign in to Docker Desktop
 - Launch and sign in to Docker Desktop for the first time
 
-Docker doesn't automatically mandate that developers re-launch and re-authenticate once a change has been made, so as not to disrupt your developers' workflow.
+So as not to disrupt your developers' workflow, Docker doesn't automatically mandate that developers re-launch and re-authenticate once a change has been made.
```

Comment:
```suggestion
To avoid disrupting your developers' workflow,, Docker doesn't automatically require that developers re-launch and re-authenticate once a change has been made.
```